suite-native/module-staking-management/src/components/StakePendingCard.tsx

```diff
@@ -0,0 +1,113 @@
+import { TouchableOpacity } from 'react-native';
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { AccountsRootState, selectAccountNetworkSymbol } from '@suite-common/wallet-core';
+import { Box, Card, Text } from '@suite-native/atoms';
+import { CryptoAmountFormatter, CryptoToFiatAmountFormatter } from '@suite-native/formatters';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    selectIsStakeConfirmingByAccountKey,
+    selectIsStakePendingByAccountKey,
+    selectTotalStakePendingByAccountKey,
+} from '@suite-native/staking';
+import { NativeStakingRootState } from '@suite-native/staking/src/types';
+
+const stakingItemStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: utils.spacings.extraSmall,
+}));
+
+const valuesContainerStyle = prepareNativeStyle(utils => ({
+    maxWidth: '40%',
+    flexShrink: 0,
+    alignItems: 'flex-end',
+    paddingLeft: utils.spacings.small,
+}));
+
+const getCardAlertProps = (isStakeConfirming: boolean, isStakePending: boolean) => {
+    if (isStakeConfirming && !isStakePending) {
+        return {
+            alertTitle: <Translation id="staking.stakePendingCard.transactionPending" />,
+            alertVariant: 'loading',
+        } as const;
+    }
+    if (!isStakeConfirming && isStakePending) {
+        return {
+            alertTitle: <Translation id="staking.stakePendingCard.addingToStakingPool" />,
+            alertVariant: 'loading',
+        } as const;
+    }
+
+    return {
+        alertTitle: undefined,
+        alertVariant: undefined,
+    } as const;
+};
+
+type StakePendingCardProps = {
+    accountKey: string;
+    handleToggleBottomSheet: (value: boolean) => void;
+};
+
+export const StakePendingCard = ({
+    accountKey,
+    handleToggleBottomSheet,
+}: StakePendingCardProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const networkSymbol = useSelector((state: AccountsRootState) =>
+        selectAccountNetworkSymbol(state, accountKey),
+    );
+
+    const totalStakePending = useSelector((state: NativeStakingRootState) =>
+        selectTotalStakePendingByAccountKey(state, accountKey),
+    );
+
+    const isStakePending = useSelector((state: NativeStakingRootState) =>
+        selectIsStakePendingByAccountKey(state, accountKey),
+    );
+    const isStakeConfirming = useSelector((state: NativeStakingRootState) =>
+        selectIsStakeConfirmingByAccountKey(state, accountKey),
+    );
+
+    const cardAlertProps = useMemo(
+        () => getCardAlertProps(isStakeConfirming, isStakePending),
+        [isStakeConfirming, isStakePending],
+    );
+
+    if (!networkSymbol) return null;
```

Comment:
according to [design](https://www.figma.com/design/ylfHUP23P94yZPcVTiPTZ6/ETH-Staking?node-id=276-19317&node-type=instance&t=HFIewEFpmkjp9XcT-0) it should also return null if there is no pending or confirming balance

Reply:
fixed